packages/hoppscotch-app/components/environments/Details.vue

```diff
@@ -219,6 +226,16 @@ export default defineComponent({
         return
       }
 
+      if (!this.envList.find((env) => env.name === this.name)) {
```

Comment:
We can't use `env.name` since multiple environments can have the same name. Use `index` instead

Reply:
Oh good catch!